packages/react-tv/renderer/ReactTVFiberEntry.js

```diff
@@ -273,6 +273,7 @@ const ReactTVRenderer = {
 
     ReactTVFiberRenderer.injectIntoDevTools({
       bundleType: process.env.NODE_ENV === 'production' ? 0 : 1,
+      version: '0.4.4',
```

Comment:
Should we fetch this from `package.json`?

Reply:
Yeah didn't thought about this, good idea!